gemrb/core/SaveGameIterator.h

```diff
@@ -29,7 +29,7 @@
 
 namespace GemRB {
 
-#define SAVEGAME_DIRECTORY_MATCHER "%d - %[A-Za-z0-9- _+*#%&|()=!?':;]"
+#define SAVEGAME_DIRECTORY_MATCHER "%d - %[^\n]"
```

Comment:
They don't get included in the a-zA-Z ranges? Too bad sscanf doesn't support :alpha:, but we should block at least  / too. Also \\ to be safe.